python/kadalu_content_apis/__init__.py

```diff
@@ -86,6 +86,11 @@ def create_template(self, name, content, template_type, output_type="text", publ
         return Template.create(self, name, content, template_type, output_type, public)
 
 
+    def upload_template(self, name, file_path, template_type, output_type="text", public=False):
```

Comment:
Similar to object, name can be made optional and take file name as default.

Reply:
updated